sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/v2/V2ScanRelationPushDown.scala

```diff
@@ -83,16 +206,35 @@ object V2ScanRelationPushDown extends Rule[LogicalPlan] {
       } else {
         withFilter
       }
-
       withProjection
   }
+
+  private def getAggregateExpression(
+      resultExpressions: Seq[NamedExpression],
+      project: Seq[NamedExpression],
+      sHolder: ScanBuilderHolder): Seq[AggregateExpression] = {
+    val aggregates = resultExpressions.flatMap { expr =>
+      expr.collect {
+        case agg: AggregateExpression =>
+          replaceAlias(agg, getAliasMap(project)).asInstanceOf[AggregateExpression]
```

Comment:
since `project.forall(_.isInstanceOf[AttributeReference])`, I don't think we need to de-alias any more.